src/modules/auth/auth.service.ts

```diff
@@ -19,8 +19,8 @@ export class AuthService {
   public async verifyMessage(session: Record<string, any>, { pubKey, message, signature }: AuthInputDto): Promise<boolean> {
     if (session.message !== message) {
       throw new BadRequestException({
-        statusCode: 400,
-        message: ERROR_MESSAGES.auth.invalidMessage.message,
+        type: ErrorType.BUSINESS_ERRORS,
```

Comment:
let's create a wrapper for business error, it can be a function with one input param(errors: IError[])